moduleframework/tools/__init__.py

```diff
@@ -28,6 +28,7 @@
 class Basic(module_framework.AvocadoTest):
     """
     :avocado: enable
+    :avocado: tags=docker,fedora,rhel,tier1
     """
 
     def test(self):
```

Comment:
This could be a bit complicated. We are calling avocado run *.py in tools directory and __init__.py is executed as well.
What is this test used for? I don't see any reason

Reply:
I don't need it, this is caused by moving module linters from /usr/share/module../modulelit -> sitepackages. This is important for installation via pip an, in case, in case there is no `init` file. pip install creates empty one and this causes that run linters via *.py causes traceback, because python file is empty. 
Thats why at the begining I were against moving modulelinters to sitepackages from some well known location.